### PR TITLE
Translate 'ConFoo Montreal 2017 is looking for Ruby speakers' (ko)

### PR DESCRIPTION
**Pull request by riseshia**
@ruby/www-ruby-lang-org-i18n-ko Please Review this! XD

### CODE REVIEW

File: ko/news/_posts/2016-08-26-confoo-cfp.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: news_post
+title: "ConFoo 몬트리올 2017 에서 루비 발표자를 찾습니다"
+author: "afilina"
+translator: "riseshia"
+date: 2016-08-26 16:00:00 +0000
+lang: ko
+---
+
+![ConFoo - Developer Conference](https://confoo.ca/images/propaganda/yul2017/en/like.png){: style="border:0; float:right; margin-left:20px;" width="180" height="130"}청중의 앞에서 웹 개발에 대해 말하고 싶으신가요? ConFoo 몬트리올 2017 콘퍼런스의 [발표자 모집][1]을 하고 있습니다!  만약 루비, 데이터베이스, 자바스크립트 등 어떤 웹 개발에 관련된 주제에 대해서 길게 말하고 싶으시면 저희에게 제안을 보내주세요.
```
Comment (marocchino) [inline ko/news/_posts/2016-08-26-confoo-cfp.md]: 만약 루비, 데이터베이스, 자바스크립트 등 어떤 웹 개발에 관련된 주제에 대해서 ->
루비, 데이터베이스, 자바스크립트는 물논 웹 개발에 관한 주제라면 무엇이든

**Comment by marocchino** [comment]
LGTM!

File: ko/news/_posts/2016-08-26-confoo-cfp.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+layout: news_post
+title: "ConFoo 몬트리올 2017 에서 루비 발표자를 찾습니다"
```
Comment (yous) [inline ko/news/_posts/2016-08-26-confoo-cfp.md]: 2017 에서 → 2017에서

File: ko/news/_posts/2016-08-26-confoo-cfp.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+author: "afilina"
+translator: "riseshia"
+date: 2016-08-26 16:00:00 +0000
+lang: ko
+---
+
+![ConFoo - Developer Conference](https://confoo.ca/images/propaganda/yul2017/en/like.png){: style="border:0; float:right; margin-left:20px;" width="180" height="130"}청중의 앞에서 웹 개발에 대해 말하고 싶으신가요? ConFoo 몬트리올 2017 콘퍼런스의 [발표자 모집][1]을 하고 있습니다!  루비, 데이터베이스, 자바스크립트는 물론 웹 개발에 관한 주제라면 무엇이든 길게 말하고 싶으시면 저희에게 제안을 보내주세요.
```
Comment (yous) [inline ko/news/_posts/2016-08-26-confoo-cfp.md]: '루비' 앞의 공백이 중복되었습니다.

**Comment by yous** [comment]
I'm done. :grin:

**Comment by riseshia** [comment]
@marocchino @yous Thanks to your reviews! I just applied all reviews, so I will merge this!

**Comment by stomar** [comment]
@riseshia welcome aboard!

Please make sure to edit commit messages of squash commits appropriately; the body of 13e523b doesn't really provide any useful or necessary information and could(should) have been dropped completely.
Also, make sure to have a look at the Wiki for some general hints.

cc @marocchino

**Comment by riseshia** [comment]
Thanks @stomar
I will check wiki to be more careful! 👍

**Pull request merged**